docs/PARAMETERS.md

```diff
@@ -27,17 +27,17 @@ Following options can be added to macro files, that are read by Geat4. Example f
 
 ## Additional parameters:
 * simulate only oPs gamma:  
- `/jpetmc/material/xad/threeGammaOnly`  
+ `/jpetmc/material/threeGammaOnly`  
 * simulate only 2 gamma:  
- `/jpetmc/material/xad/twoGammaOnly`  
+ `/jpetmc/material/twoGammaOnly`  
 * simulate only pick-off process (2g & long lifetime):
- `/jpetmc/material/xad/pickOffOnly`  
-* define longest lifetime component:  
- `/jpetmc/material/xad/oPslifetime`  
-* define fraction of oPs annihilations (0-1):  
- `/jpetmc/material/xad/oPsFraction`  
-* define fraction of pick-off processes (0-1):  
- `/jpetmc/material/xad/pickOffFraction`  
+ `/jpetmc/material/pickOffOnly`  
+* adding oPs component for XAD4
+ `/jpetmc/material/xad4/oPsComponent (mean lifetime in ns) (probability 0-100)`  
```

Comment:
Should it be probabliity from 0 to 1? Not 100?

Reply:
Changed to 100 %. I think it is easier for users to use percents